tooling/autodispose-rx-error-prone-checker/build.gradle

```diff
@@ -0,0 +1,70 @@
+plugins {
+  id "net.ltgt.errorprone" version "0.0.13"
+  id "com.github.johnrengelman.shadow" version "2.0.1"
+  id "com.github.johnrengelman.plugin-shadow" version "2.0.0"
+  id "java"
+}
+
+// we use this config to get the path of the JDK 9 javac jar, to
+// stick it in the bootclasspath when running tests
+configurations.maybeCreate("epJavac")
```

Comment:
@kageiit can you confirm this is correct?

Reply:
actually if you wouldn't mind taking a general review of this config, I'd appreciate the extra set of eyes

Reply:
https://github.com/uber/AutoDispose/pull/156/commits/05a3a34b3778da0031e5dfa0161256f224ed6cdf